build.sbt

```diff
@@ -9,20 +9,20 @@ crossScalaVersions := Nil // crossScalaVersions must be set to Nil on the aggreg
 lazy val bench = project
   .dependsOn(
     `weepickle-tests` % "compile;test",
-    `weejson-upickle`,
   )
   .enablePlugins(JmhPlugin)
   .settings(
     noPublish,
     crossScalaVersions := Seq(scala213, scala3),
     libraryDependencies ++= Seq(
+      "com.lihaoyi" %% "upickle" % "3.1.0", // need the latest Scala 3 support
       "com.fasterxml.jackson.core" % "jackson-databind" % "2.13.0",
```

Comment:
Should we pump the version of this (and any others?) 2.13.0 has 4 known vulnerabilities

Reply:
Sure.

Reply:
Take a look at https://github.com/rallyhealth/weePickle/pull/121/commits/b5705fcb4ee0549b0d19665aac0a8985dfdf994a. I did what I could (and still stay binary compatible).